aggregator/map.go

```diff
@@ -123,6 +123,10 @@ func (m *metricMap) Tick(target time.Duration) tickResult {
 	}
 }
 
+func (m *metricMap) Close() {
+	m.metricLists.Close()
```

Comment:
No need to protect against multiple or concurrent close of this list?

Reply:
This is already handled in `metricsLists.Close`. It's safe to call it twice with the second call being a no op.